### ISSUE TITLE
Doesn't detect `pdb.set_trace()`

### ISSUE THREAD

**Issue by aviv** [vyahello/flake8-debug#6]
Hi there — just a heads up that this will detect

```
from pdb import set_trace

set_trace()
```

but not:

```
import pdb

pdb.set_trace()
```

**Comment by vyahello** [comment]
@aviv released new `0.2.0` version of the plugin, your changes are already in, feel free to update package via pip

thanks

**Comment by aviv** [comment]
Thanks a bunch @vyahello! Much appreciated.
